cfpb/cfgov-refresh
# Ask CFPB: Clean up unused portions of ask pages

## Removals

- Remove unused ask category page.
- ~~Remove references to nonexistent `ask-search-bar.html` template in .po file.~~
- Remove empty `content` section of Ask results page.

## Testing

1. Pull branch, run migrations.
2. Create a new Ask Results Page type and see that there is only a title, no empty content area.

## Screenshots

Before:
<img width="1233" alt="Screen Shot 2019-09-11 at 12 42 14 PM" src="https://user-images.githubusercontent.com/704760/64718405-1c816080-d494-11e9-828d-6940cd55a186.png">

After:
<img width="1237" alt="Screen Shot 2019-09-11 at 12 57 04 PM" src="https://user-images.githubusercontent.com/704760/64718411-2014e780-d494-11e9-8f26-d97df7b7cbb1.png">
